pageserver/src/http/routes.rs

```diff
@@ -661,6 +667,101 @@ async fn tenant_config_handler(mut request: Request<Body>) -> Result<Response<Bo
     json_response(StatusCode::OK, ())
 }
 
+#[cfg(any(feature = "testing", feature = "failpoints"))]
```

Comment:
NIT: I wonder, if we should just merge both features into one, does not appear to me that we'll ever use them separately.

Reply:
Makes sense - updated the rest of the locations using `#[cfg(feature = "failpoints")]` or `--features failpoints`. Also simplified the testing api route macro (renamed `cfg_disabled!` -> `testing_api!`)